include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -40,34 +32,30 @@ class AvailabilityChecksManager {
     ~AvailabilityChecksManager() = default;
 
     /// @brief Checks if a certain Vulkan instance layer is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceLayerProperties.html
     /// @param instance_layer_name The name of the Vulkan instance layer.
-    /// @return true if the Vulkan instance layer is available, false otherwise.
+    /// @return 'true' if the Vulkan instance layer is available, 'false' otherwise.
```

Comment:
```suggestion
    /// @return ``true`` if the Vulkan instance layer is available, ``false`` otherwise.
```
Use ` `` `, as this will be converted into the mono notes you know from here `mono`. Remember that all resStructrured Text stuff is available!


Also i think you can remove the `false otherwise`.

Reply:
Done. I removed the `false otherwise` everywhere.